monai/visualize/utils.py

```diff
@@ -47,15 +48,19 @@ def matshow3d(
     Create a 3D volume figure as a grid of images.
 
     Args:
-        volume: 3D volume to display. Higher dimensional arrays will be reshaped into (-1, H, W).
+        volume: 3D volume to display. data shape can be `BCHWD`, `CHWD` or `HWD`.
+            Higher dimensional arrays will be reshaped into (-1, H, W, [C]), `C` depends on `channel_dim` arg.
             A list of channel-first (C, H[, W, D]) arrays can also be passed in,
             in which case they will be displayed as a padded and stacked volume.
         fig: matplotlib figure to use. If None, a new figure will be created.
         title: title of the figure.
         figsize: size of the figure.
         frames_per_row: number of frames to display in each row. If None, sqrt(firstdim) will be used.
-        frame_dim: for higher dimensional arrays, which dimension (`-1`, `-2`, `-3`) is moved to the `-3`
+        frame_dim: for higher dimensional arrays, which dimension (`-1`, `-2`, `-3`) is moved to the `-3`.
             dim and reshape to (-1, H, W) shape to construct frames, default to `-3`.
+        channel_dim: if not None, explicitly specify the channel dimension, which will be switched to the end,
```

Comment:
```
channel_dim: if not None, explicitly specify the channel dimension to be transposed to the last dimension.
            This can be used to plot RGB color image. If None, the channel dimension will be reshaped with `frame_dim`
            and `batch_dim`. Note that it can only support 3D input image. Default is None.
```
I've rewritten this here but it's not fully clear what is going on the None case where frame_dim and batch_dim are used, Do you mean that frame_dim is moved to the end?

Reply:
Hi @ericspod , for the None case, it's same as previous behavior which will flatten the `batch dim`, `channel dim`, `frame dim` together in `reshape(-1, image.shape[-2], image.shape[-1])`.
Let me clarify in the doc-string.

Thanks.

Reply:
updated in the latest commit.

Thanks.